app/assets/stylesheets/custom.css.scss

```diff
@@ -189,3 +196,45 @@ a:hover, a:active {
     background-color: $lighter-crab-red;
     border-color: $lighter-crab-red;
 }
+
+table {
+   width: 75%;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered margin-top, width